Secret-agency-persistence/src/main/cz/fi/muni/pa165/secretagency/entity/Mission.java

```diff
@@ -0,0 +1,129 @@
+package main.cz.fi.muni.pa165.secretagency.entity;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+/**
+ * Mission entity for Secret agency project.
+ *
+ * @author Adam Skurla (487588)
+ */
+public class Mission {
+    private Long id;
+    private Double[] location;
+    private String missionType;
```

Comment:
Neuděláme pro missionType vlastní Enum? Přijde mi to jako lepší řešení, než je String, jelikož do Stringu lze zapsat jakýkoliv text.  missionType je podle mě výčet hodnot, kam by se neměly přidávat jiné hodnoty, než budou v daném Enumu.